### ISSUE TITLE
Lottery button dynamic effect 

### ISSUE THREAD

**Issue by ryan8614** [ryan8614/5505-Project#5]
<div class="lottery-container col-sm-6">
                <div class="spin-wheel">
                    <div class="one">1</div>
                    <div class="two">2</div>
                    <div class="three">3</div>
                    <div class="four">4</div>
                    <div class="five">5</div>
                    <div class="six">6</div>
                </div>
                <span class="wheel-center"></span>
                <button type="button" id="spin" class="btn btn-primary">Spin</button>
                <div class="stopper"></div>
            </div>

            
            
            <button type="button" id="raffleButton" class="btn btn-primary mt-5">Try Your Luck!</button>
            
            <div class="modal" id="raffleModal">
                <div class="modal-dialog">
                    <div class="modal-content">
                        
                        <div class="modal-header">
                            <h4 class="modal-title">Congratulations!</h4>
                            <button type="button" class="btn-close" data-bs-dismiss="modal"></button>
                        </div>
                        
                        <div class="modal-body">
                            
                            <p>You've won a piece!</p>
                            <img id="winningSplitImage" src="" alt="Winning Split" class="img-fluid"> 
                            <p id="winningSplitName"></p>
                        </div>
                    </div>
                </div>
            </div>

You need to combine these two buttons through js, and you can create relevant js files to achieve this dynamic loading effect.

**Comment by JasAAA** [comment]
yeah,I have tried to do so,but there is some confilcts affects js functions.I haven't figure out yet.So I temperarily seperate this two functions.

**Comment by Shaohongzheng** [comment]
all good. i will try to fix them.
